magenta/models/latent_transfer/configs/joint_exp_mnist2wavegan_parameterized.py

```diff
@@ -16,9 +16,8 @@
 
 # pylint:disable=invalid-name
 
-from functools import partial
-
-import temsorflow as tf
+import functools
```

Comment:
What you've done is fine for pylint, but another way to do this that allows you to keep things simple is 

```
import functools

...

partial = functools.partial
```

Reply:
good point, but i'm going to leave as-is unless there is a major objection